app/views/complaints/_complaints_table_detail.html.erb

```diff
@@ -1,22 +1,23 @@
+<% complaint = Complaint.new(complaint_data) %>
+
 <tr>
-  <% formatted_status = status(complaint[:attributes]) %>
   <th scope="row" role="rowheader">
-    <span class="<%= formatted_status == "New" ? "text-bold" : "" %>">
-      <%= link_to complaint[:id], complaint_path(complaint[:id]), class: "usa-link" %>
+    <span class="<%= complaint.new? ? "text-bold" : "" %>">
+      <%= link_to complaint.id, complaint_path(complaint.id), class: "usa-link" %>
     </span>
   </th>
   <td
-    data-sort-value="<%= DateTime.parse(complaint[:attributes][:creationDate]).to_i %>"
+    data-sort-value="<%= sort_date(complaint.creation_date) %>"
```

Comment:
[thought] i'm thinking about whether there'd be drawbacks to just putting sort_date and sort_status on the complaint object. it's view-centric, so i get why it's in a helper. this is out of scope, but it would be a good opportunity for a view decorator!

Reply:
I don't know much about view decorators to weigh in on that aspect, but it definitely seems like there's probably a better approach to this